### PR TITLE
Add core files.

### PR DESCRIPTION
**Pull request by jeiholzer**
please review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "bedrock-angular-forge",
+  "version": "1.0.0",
```
Comment (mattcollier) [inline package.json]: version on implementation branches needs to be 0.0.1-0

The reason for this is that the release system expects the version to be below v1 when we do a 1.0.0 release.

**Review by mattcollier — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (jeiholzer) [reply package.json]: wondered about that after seeing a different module, should have asked

**Review by mattcollier — APPROVED** [review]

**Comment by mattcollier** [comment]
@jeiholzer please squash once more.

**Comment by mattcollier** [comment]
@jeiholzer you did already, nvm :)

**Pull request merged**